lib/Runtime/Debug/DebugContext.cpp

```diff
@@ -260,7 +260,7 @@ namespace Js
             return false;
         });
 
-        if (this->scriptContext != nullptr && !this->scriptContext->IsClosed())
+        if (SUCCEEDED(hr) && this->scriptContext != nullptr && !this->scriptContext->IsClosed())
```

Comment:
So what is the idea now? if we fail on one we return the failures to host? but we have changed our scriptcontext already - so this is more unstable.